.claude/scripts/mcp_common.sh

```
@@ -0,0 +1,2130 @@
+            log_with_timestamp "Parallel installation PROCESS_FAILED: $server_name"
+        fi
+
+        # Cleanup temp files
+        if [ -n "$result_file" ]; then
+            rm -f "$result_file"
+        fi
+        if [ -n "$log_file" ]; then
+            rm -f "$log_file"
+        fi
+        set +u
+        unset PARALLEL_RESULT_FILES["$server_name"]
```

Shellcheck (0.11.0)

[warning] Quote arguments to unset so they're not glob expanded.

(SC2184)